content/security/docs/iam.md

```diff
@@ -829,9 +829,21 @@ print("---")
 print("sqs response:")
 print(sqsresponse)
 ```
-
 If you're migrating an application from another AWS compute service, such as EC2, to EKS with IRSA, this is a particularly important detail. On other compute services initializing an AWS SDK session does not call AWS STS unless you instruct it to.
 
+
```

Comment:
Can we add some wording around whats the recommendation, regional vs gloabl STS endpoint and place this part under this section `Controlling Access to EKS Clusters`. 

Also, seems like the command mentioned in this section `Controlling Access to EKS Clusters` is missing the --region flag, we should update the sample command and the output from this command.